dbms/src/Flash/ResourceControl/MockLocalAdmissionController.h

```diff
@@ -30,22 +30,18 @@ inline uint64_t nopGetPriority(const std::string &)
 {
     return 1.0;
 }
-inline bool nopIsResourceGroupThrottled(const std::string &)
-{
-    return false;
-}
-
 
 // This is only for ResourceControlQueue gtest.
 class MockLocalAdmissionController final : private boost::noncopyable
 {
 public:
-    MockLocalAdmissionController()
+    explicit MockLocalAdmissionController(bool enable = true)
```

Comment:
If the overhead of enable by default is not large, maybe this bool variable may not be needed.